### PR TITLE
Adding Drift Page

### PR DESCRIPTION
**Pull request by johnsonm325**

### CODE REVIEW

File: src/SmartComponents/DriftPage/DriftPage.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { Main, PageHeader, PageHeaderTitle } from '@red-hat-insights/insights-frontend-components';
+import { Card, CardBody } from '@patternfly/react-core';
+
+import { DriftTable } from './DriftTable/DriftTable';
+
+export class DriftPage extends Component {
```
Comment (ehelms) [inline src/SmartComponents/DriftPage/DriftPage.js]: Try switching this to export default

Comment (johnsonm325) [reply src/SmartComponents/DriftPage/DriftPage.js]: It was 2 issues. Needed to:
```
export default DriftPage;
```
and needed to:
```
import DriftTable from './DriftTable/DriftTable';
```

File: src/Routes.js [hunk 2]
```diff
@@ -4,7 +4,8 @@ import React from 'react';
 
 import asyncComponent from './Utilities/asyncComponent';
 
-const DriftTable = asyncComponent(() => import('./SmartComponents/DriftPage/DriftTable/DriftTable'));
+//const DriftTable = asyncComponent(() => import('./SmartComponents/DriftPage/DriftTable/DriftTable'));
```
Comment (ehelms) [inline src/Routes.js]: Can be dropped

Comment (johnsonm325) [reply src/Routes.js]: ahhh!

**Pull request merged**